src/CMakeLists.txt

```diff
@@ -68,8 +68,8 @@ else ()
     qt5_add_translation(qmFiles ${tsFiles})
 endif ()
 
-file(GLOB HEADERS *.h)
-file(GLOB SOURCES *.cpp)
+file(GLOB_RECURSE HEADERS *.h)
+file(GLOB_RECURSE SOURCES *.cpp)
```

Comment:
generally speaking, GLOB_RECURSE isn't always a great idea, especially when you want to have control on which file really gets compiled. Listing the files singularly allows you to get more fine-grained details on the file being added to the compilation unit.

Reply:
Yeah... I need a better plan... I will make more of these "modules" and it would probably be better to put a CMakeLists.txt in the src/licenses directory

Reply:
I'll definitely do something about this, but not in this review - it's big enough as it is :sweat_smile: